src/adapters/wallet/secondary/KeplrWalletGateway.ts

```diff
@@ -86,7 +86,8 @@ export class KeplrWalletGateway implements Wallet {
       ) ?? {}
   }
 
-  public readonly isAvailable = (): boolean => !!window.keplr && !!window.keplr.getOfflineSigner
+  public readonly isAvailable = (): boolean =>
+    typeof window !== 'undefined' && !!window.keplr && !!window.keplr.getOfflineSigner
```

Comment:
Why not create an explicit predicate named `isBrowser()` or `isSSR` (something meaningful) that tells if `window` is available or not? (since this test being done in several places).